lms/djangoapps/certificates/generation_handler.py

```diff
@@ -255,14 +255,14 @@ def is_using_certificate_allowlist(course_key):
     return CERTIFICATES_USE_ALLOWLIST.is_enabled(course_key)
 
 
-def _is_using_v2_course_certificates(course_key):
+def is_using_v2_course_certificates(course_key):
```

Comment:
Dropped the leader `_` as the function is no longer private to `generation_handler.py`.

Reply:
Yep, this makes sense. 

Could you switch _certs.py_ to using the method as imported from _api.py_, just to keep us moving toward the goal of other django apps using the API methods?

Ex. https://github.com/edx/edx-platform/blob/master/lms/djangoapps/certificates/api.py#L198

Reply:
Yes! Thanks for calling this out. I should have realized this was in another django app. Woops.